### PR TITLE
Cosmetic changes

### PR DESCRIPTION
**Pull request by yaronvel**
1. Added links to block explorers for each account.
2. Removed annoying animation when displaying the donations sum.

### CODE REVIEW

File: donate.slim [hunk 1]
```diff
@@ -67,20 +70,20 @@ html lang="en"
...
-                p.progress#total-fund-raised.total-fund-raised.odometer 0.00
+                p.progress#total-fund-raised.total-fund-raised 0.00
                 p.progress ETH
           div.donation-instruction
             h3 Our addresses
             div.instruction-wrapper
               div.address
                 p.label.ethereum Ethereum:
-                p.address #{smart_pool_address}
+                p.address <a href=#{smart_pool_ethereum_url}>#{smart_pool_address}</a>
```
Comment (tranvictor) [inline donate.slim]: I think we should better eliminate pure HTML in slim, it's shorter
```
p.address
  a href="#{smart_pool_ethereum_url}" #{smart_pool_address}
```

File: index.slim [hunk 2]
```diff
@@ -110,7 +110,7 @@ html lang="en"
             div.border
               h3.donation-title #{donation_title}
               p.progress Current donated amount:
-              p.progress#total-fund-raised.total-fund-raised.odometer 0.00
+              p.progress#total-fund-raised.total-fund-raised 0.00
```
Comment (tranvictor) [inline index.slim:113]: We can also remove corresponding javascript and stylesheet files to odometer if we want to remove animation.

**Review by tranvictor — CHANGES_REQUESTED** [review]

File: donate.slim [hunk 3]
```diff
@@ -67,20 +70,20 @@ html lang="en"
               div.border
                 h3.donation-title #{donation_title}
                 p.progress Current donated amount:
-                p.progress#total-fund-raised.total-fund-raised.odometer 0.00
+                p.progress#total-fund-raised.total-fund-raised 0.00
```
Comment (tranvictor) [inline donate.slim]: Should we also change the `0.00` to `loading...` or some sort?

File: index.html [hunk 4]
```diff
@@ -56,7 +56,7 @@ <h3 class="donation-title">
               <p class="progress">
                 Current donated amount:
               </p>
-              <p class="progress total-fund-raised odometer" id="total-fund-raised">
+              <p class="progress total-fund-raised" id="total-fund-raised">
                 0.00
```
Comment (tranvictor) [inline index.html:60]: Change `0.00` to `loading...` here too?

File: donate.slim (hunk 1 above)
Comment (yaronvel) [reply donate.slim]: It doesn't work.
I get
```
Ethereum: a href="https://etherscan.io/address/0x98F62d8aD5a884C8bbcf262591DFF55DAb263B80" 0x98F62d8aD5a884C8bbcf262591DFF55DAb263B80
``` 
in the webpage.

**Review by tranvictor — APPROVED** [review]

**Pull request merged**